src/hooks/useOpenApp.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { renderHook, act } from '@testing-library/react-hooks'
+
+import { useOpenApp, openApp, closeApp } from './useOpenApp'
+
+test('switches between closed and opened states', () => {
```

Comment:
This test is a bit difficult to understand. I had to read the `useOpenApp` code to understand that we expect a class to appear and disappear

Can't we use enum values here?